lib/xml/parser.js

```diff
@@ -121,39 +121,71 @@ function parseEvents(xml) {
                         // Handle this next expanded occurence
                         const nextEvent = icalEvent.getOccurrenceDetails(nextOccuranceTime);
 
-                        console.log("==> ", nextOccuranceTime.toString());
+                        data = {};
 
                         if (modifiedOccurences.length === 0) {
                             // No events have been modified
+                            const dtstart = nextEvent.startDate;
+                            const dtend = nextEvent.endDate;
+
                             data.title = nextEvent.item.summary;
                             data.uid = nextEvent.item.uid;
                             data.location = nextEvent.item.location;
                             data.description = nextEvent.item.description;
-                            data.start = nextEvent.startDate.toString();
-                            data.end = nextEvent.endDate.toString();
-                            data.createdAt = vevent.getFirstPropertyValue("created").toICALString();
+                            data.start = new Date(dtstart);
```

Comment:
I think it would look nicer if you create an object within the if and and else branch. So you do not have to repeat to write `data` 9 times. 

```javascript

// can we use es2015 here?
const { summary: title, uid, location, //...} = nextEvent.item;

if (modifiedOccurences.length === 0) {
   aVarWithBetterNaming = {
    title,
   uid,
   // ...
  }
}

```